recipes/minvar/licence.md

```diff
@@ -0,0 +1,146 @@
+SOFTWARE LICENSE AGREEMENT (hereinafter referred to as AGREEMENT)
+=================================================================
+
+This is a legal agreement between you (hereinafter referred to as the LICENSEE)
+and the University of Zurich, Rämistrasse 71, 8006 Zurich (hereinafter referred
+to as UNIVERSITY OF ZURICH) pertaining to the right to use the software product
+“MinVar” (hereinafter referred to as SOFTWARE). Using the SOFTWARE indicates
+your acceptance of these terms and conclusion of the Agreement between the
```

Comment:
This sounds alright, but normally this license file is part of the tarball/package and license_file with copy it by them self. So no need to put it into the repo.